site/connections.md

```diff
@@ -0,0 +1,261 @@
+ * [Flow control](#flow-control)
+
+and other topics related to connections.
+
+## <a id="basics" class="anchor" href="#basics">The Basics</a>
+
+Applications interact with RabbitMQ using client libraries. There are [client libraries](/devtools.html)
+available for many programming languages and platforms.
+Each protocol has its own set of client libraries. Most client libraries are open source.
+
+Both client libraries and applications that use them are referred to as "clients" in this guide.
+Where the difference matters, a more specific term is used (e.g. "application").
+
+All protocols supported by RabbitMQ are TCP-based and assume long-lived connections (a new connection
+is not opened per protocol operation) for efficiency. One client library connection uses a single
+TCP connection. In for a client to successfully connect, target RabbitMQ node must allow for
+connections on a [certain protocol-specific port](/networking.html).
+
+After a client [connects](#lifecycle) and successfully authenticates with a RabbitMQ node, it can
+publish and consume messages, define topology and perform other operations that are provided in the protocol
+and supported both by the client library and the target RabbitMQ node.
+
+Since connections are meant to be long-lived, clients usually [consume messages](/consumers.html) by registering
+a subscription and having messages delivered (pushed) to them instead of polling.
+
+When a connection is no longer necessary, applications must close them to conserve resources.
+Apps that fail to do it run the risk of eventually exhausting its target node of resources.
+
+Operating systems have a [limit around how many TCP connections (sockets) a single process can have open](/networking.html#open-file-handle-limit)
+simultaneously. The limit is often sufficient for development and some QA environments.
+[Production environments](/production-checklist.html) must be configured to use a higher limit in order to support
+a larger number of concurrent client connections.
+
+### Protocol Differences
+
+Different messaging protocols use different ports. Ports also vary for plain TCP and TLS-enabled connections.
+The [Networking guide](/networking.html#ports) covers all ports used by RabbitMQ depending on what protocols are enabled, whether TLS
+is used and so on.
+
+#### AMQP 0-9-1
+
+AMQP 0-9-1 provides a way for connections to multiplex over a single TCP connection. That means an application
+can open multiple "lightweight connections" called channels on a single connection. AMQP 0-9-1 clients open one or more
+channels after [connecting](#lifecycle) and perform protocol operations (manage topology, publish, consume)
+on the channels.
+
+#### AMQP 1.0
+
+AMQP 1.0 provides a way for connections to multiplex over a single TCP connection. That means an application
+can open multiple "lightweight connections" called sessions on a single connection.
+Applications that set up one or more links to publish and consume messages.
```

Comment:
I'm not sure this sentence is correct, I don't understand it.

Reply:
It should have been "then" instead of "that".